### ISSUE TITLE
Path path information for background image loaded from filesystem is discarded and image can't be displayed.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jkoo7yeha/dompdf#183]
```
What steps will reproduce the problem?

Using dompdf-0.5.1: Load HTML from the filepath and within a CSS file reference 
image from another directory.

In app/template/print.html:

    <link rel="stylesheet" type="text/css" href="stylesheets/invoice.css" />

In app/stylesheets/invoice.css:

    .paid {
        background-image: url('../images/invoice/paid.png');
    }

In app/index.php:

    $dompdf = new DOMPDF();
    $dompdf->load_html(file_get_contents('template/print.html'));
    $dompdf->set_base_path(dirname(__FILE__));
    $dompdf->render();


What is the expected output? What do you see instead?

DOMPDF tries to load app/stylesheets/paid.png instead of 
app/stylesheets/../images/paid.png. 

No work arounds in my own code seemed to fix this. Style::set_background() 
tries to resolve the absolute path by running filename through realpath() but 
Image_Cache::resolve_url() expects the filename to be relative to base_path.


What version of the product are you using? On what operating system?

Windows 7, dompdf-0.5.1


Please provide any additional information below.

The following fix leaves relative path information intact.

--- style.cls.php   (revision 275)
+++ style.cls.php   (revision 276)
@@ -1181,7 +1181,9 @@
     // Resolve the url now in the context of the current stylesheet
     $parsed_url = explode_url($val);
     if ( $parsed_url["protocol"] == "" && $this->_stylesheet->get_protocol() == "" )
-      $url = realpath($this->_stylesheet->get_base_path() . 
$parsed_url["file"]);
+      //jduck - Patched the following line. Previous line could not handle
+      //images outside of the stylesheet path for stylesheets loaded from 
filesystem.
+      $url = $parsed_url["path"] . $parsed_url["file"];
     else
       $url = build_url($this->_stylesheet->get_protocol(),
                        $this->_stylesheet->get_host(),


```

Original issue reported on code.google.com by `joshd...@gmail.com` on 20 Sep 2010 at 5:01

**Comment by GoogleCodeExporter** [comment]
```
This appears to already have been addressed in the 0.6.0 code base:
http://code.google.com/p/dompdf/source/browse/trunk/dompdf/include/style.cls.php
#1326

```

Original comment by `eclecticgeek` on 21 Sep 2010 at 5:47
- Changed state: **Addressed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `eclecticgeek` on 30 May 2013 at 5:15
- Added labels: **Restrict-AddIssueComment-Commit**
